vendor/bat-native-ledger/src/bat_get_media_unittest.cc

```diff
@@ -179,4 +179,71 @@ TEST(BatGetMediaTest, GetYoutubeUserFromUrl) {
   ASSERT_EQ(user, "brave");
 }
 
+TEST(BatGetMediaTest, getRealEnteredYTPath) {
```

Comment:
please add test for
- empty string
- just `/`
- and string without `/` like `bravesoftware/videos`

Reply:
Updated. As a side note: we don't get `path` without a leading `/`. If we did, this logic would need to change.

Reply:
sounds good